http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala

```diff
@@ -70,10 +70,10 @@ class ConfigReaderSpec extends AnyFlatSpec with Matchers {
               "2.1",
               true,
               List(
-                "https://raw.githubusercontent.com/DataBiosphere/leonardo/270bd6aad916344fadc06d1a51629c432da663a8/http/src/main/resources/init-resources/azure_vm_init_script.sh"
+                "https://raw.githubusercontent.com/DataBiosphere/leonardo/61f9ad246382ade9a5242e4c6f899633876767f7/http/src/main/resources/init-resources/azure_vm_init_script.sh"
               )
             ),
-            "terradevacrpublic.azurecr.io/terra-azure-relay-listeners:48a3a7b",
+            "terradevacrpublic.azurecr.io/terra-azure-relay-listeners:decc8b0",
```

Comment:
This is the main part that needs to be reverted to a correct version.

`decc8b0` will not include some of andrew's changes.
https://portal.azure.com/#view/Microsoft_Azure_ContainerRegistries/RepositoryBlade/id/%2Fsubscriptions%2Fc5f8eca3-f512-48cb-b01f-f19f1af9014c%2FresourceGroups%2Facr_resource_group%2Fproviders%2FMicrosoft.ContainerRegistry%2Fregistries%2FterraDevAcrPublic/repository/terra-azure-relay-listeners

This is the repository that holds all of ouer terra-azure-relay-listener versions.

You'll need a revert PR on there, and put the hash from that build into this line.

Reply:
This needs to be reverted
https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/29